addons/gdUnit4/src/coverage/GdUnitTestCoverageChecker.gd

```diff
@@ -0,0 +1,765 @@
+class_name GdUnitTestCoverageChecker
+extends RefCounted
```

Comment:
I do not want to provide a gdunit4-specific version, as this would increase the maintenance effort.
Please refer to the original coverage code of https://github.com/jamie-pate/godot-code-coverage

Reply:
Hi, can you please push your [godot-code-coverage](https://github.com/jamie-pate/godot-code-coverage) static type fixes to the original repo?

Reply:
@emabrey you still working on?

Reply:
I couldn't get it working unfortunately.

Reply:
But you are showing a running version here?
What is blocking you?